packages/components/molecules/f-rating/src/components/Rating.vue

```diff
@@ -34,13 +101,27 @@ export default {
 @use '@justeat/fozzie/src/scss/fozzie' as f;
 
 .c-rating {
-    display: flex;
-    justify-content: center;
-    min-height: 80vh;
-    width: 80vw;
-    margin: auto;
-    border: 1px solid red;
-    font-family: f.$font-family-base;
-    @include f.font-size(heading-m);
+    .c-rating-star-wrapper {
```

Comment:
Looks like the nesting is still here – can just move all the classes to be individually specified (maybe drop me a message if it helps - not the easiest thing to explain sometimes in comments!)